packages/backend/src/tenants/settings/service.test.ts

```diff
@@ -371,4 +373,37 @@ describe('TenantSetting Service', (): void => {
       expect(result).toEqual([])
     })
   })
+
+  describe('get settings by value', (): void => {
+    test('can get settings by wallet address prefix setting', async (): Promise<void> => {
```

Comment:
we should add a negative test to check that we avoid fetching a tenant if the prefix doesn't match